packages/yumemi_lints/pubspec.yaml

```diff
@@ -6,6 +6,17 @@ repository: https://github.com/yumemi-inc/flutter-yumemi-lints
 environment:
   sdk: '>=2.17.0 <4.0.0'
 
+dependencies:
+  args: ^2.3.1
+  file: ^6.1.4
+  meta: ^1.7.0
```

Comment:
![memo-badge](https://img.shields.io/badge/review-memo-lightgrey) 
meta version lowered to ^1.7.0 for flutter v3.0.0 project compatibility